forward_script/src/PageInformationCollector.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace Wikimedia\ForwardScript;
+
+use Mediawiki\Api\MediawikiApi;
+use Mediawiki\Api\SimpleRequest;
+use Symfony\Component\Process\Process;
+use Symfony\Component\Process\Exception\RuntimeException;
+
+/**
+ * Get page content and categories from Wikipedia, return information about the page.
+ *
+ * @package Wikimedia\ForwardScript
+ */
+class PageInformationCollector {
+
+	/**
+	 * @var MediawikiApi
+	 */
+	private $api;
+
+	/**
+	 * @var Process
+	 */
+	private $processCommand;
+
+	/**
+	 * @var array
+	 */
+	private $defaultCategories;
+
+	public function __construct( MediawikiApi $api, Process $processCommand, $defaultCategories=[] ) {
```

Comment:
You might also add a type hint for the `$defaultCategories`. Or is there some reason why it is not there?